tfx/components/tuner/component_test.py

```diff
@@ -81,6 +81,18 @@ def testConstructDuplicateUserModule(self):
           module_file='/path/to/module/file',
           tuner_fn='path.to.tuner_fn')
 
+  def testConstructWithCustomSplits(self):
+    self.train_args = trainer_pb2.TrainArgs(splits=['train'], num_steps=100)
+    self.eval_args = trainer_pb2.EvalArgs(splits=['eval'], num_steps=50)
```

Comment:
you can just update line37&38 and drop this test, same for the Trainer

this is because we didn't do any special handling in component.py